From dce9ef7db6b12bcdee8a823ce2adb51c6d495d39 Mon Sep 17 00:00:00 2001
From: Alexander Martinz <amartinz@shiftphones.com>
Date: Tue, 19 Mar 2019 18:53:03 +0100
Subject: [PATCH 3/6] LightSettingsDialog: remove unused OnColorChangedListener

We never set the listener but use it, wrapped with null checks.
Remove it.

Change-Id: I223ab27b6892c5809389a936a6ed7ea768bc3ae4
Signed-off-by: Alexander Martinz <amartinz@shiftphones.com>
---
 .../notificationlight/LightSettingsDialog.java         | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/src/org/lineageos/lineageparts/notificationlight/LightSettingsDialog.java b/src/org/lineageos/lineageparts/notificationlight/LightSettingsDialog.java
index 3382580..ba6c6b8 100644
--- a/src/org/lineageos/lineageparts/notificationlight/LightSettingsDialog.java
+++ b/src/org/lineageos/lineageparts/notificationlight/LightSettingsDialog.java
@@ -50,7 +50,6 @@ import org.lineageos.internal.notification.LedValues;
 import org.lineageos.internal.notification.LightsCapabilities;
 import org.lineageos.internal.notification.LineageNotification;
 import org.lineageos.lineageparts.R;
-import org.lineageos.lineageparts.notificationlight.ColorPickerView.OnColorChangedListener;
 
 import java.util.ArrayList;
 import java.util.Locale;
@@ -74,8 +73,6 @@ public class LightSettingsDialog extends AlertDialog implements
     private Spinner mPulseSpeedOff;
     private LayoutInflater mInflater;
 
-    private OnColorChangedListener mListener;
-
     private NotificationManager mNotificationManager;
 
     private boolean mReadyForLed;
@@ -244,10 +241,6 @@ public class LightSettingsDialog extends AlertDialog implements
         mNewColor.setColor(color);
         mHexColorInput.setText(String.format(Locale.US, format, color & mask));
 
-        if (mListener != null) {
-            mListener.onColorChanged(color);
-        }
-
         updateLed();
     }
 
@@ -467,9 +460,6 @@ public class LightSettingsDialog extends AlertDialog implements
                 mColorPicker.setColor(color);
                 mNewColor.setColor(color);
                 updateLed();
-                if (mListener != null) {
-                    mListener.onColorChanged(color);
-                }
             } catch (IllegalArgumentException ex) {
                 // Number format is incorrect, ignore
             }
-- 
2.17.1

